net/openelp/Makefile

```diff
@@ -0,0 +1,64 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=openelp
+PKG_VERSION:=0.9.2
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/cottsay/openelp/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=1427a2fe6f22856a66b9c687d7f14b7915abeeed64951ae596084f9366ec4256
+
+PKG_MAINTAINER:=Scott K Logan <logans@cottsay.net>
+PKG_LICENSE:=BSD-3-Clause
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_BUILD_DEPENDS:=pcre2
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+define Package/openelp
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=Open Source EchoLink Proxy
+  URL:=https://github.com/cottsay/openelp#readme
+  DEPENDS:=+libpcre2
+  ABI_VERSION:=0
+endef
+
+define Package/openelp/description
+  OpenELP is an open source EchoLink proxy for Linux and Windows. It aims
+  to be efficient and maintain a small footprint, while still implementing
+  all of the features present in the official EchoLink proxy.
+
+  OpenELP also has the ability to bind to multiple network interfaces
+  which are routed to unique external IP addresses, and therefore is
+  capable of accepting connections from multiple clients simultaneously.
+endef
+
+CMAKE_OPTIONS += \
+  -DSYSCONF_INSTALL_DIR=/etc
+
+define Package/openelp/conffiles
+  /etc/ELProxy.conf
+endef
+
+define Package/openelp/install
+	$(INSTALL_DIR) $(1)/etc
+	$(INSTALL_CONF) $(PKG_INSTALL_DIR)/etc/ELProxy.conf $(1)/etc/
+	$(INSTALL_DIR) $(1)/usr/lib
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libopenelp.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/openelpd $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/etc/init.d
+	$(INSTALL_BIN) $(CURDIR)/files/openelp.init $(1)/etc/init.d/openelp
+endef
+
+define Build/InstallDev
```

Comment:
You can use the default cmake `Build/InstallDev` (instead of defining your own) if you set `CMAKE_INSTALL:=1` (I suggest where `PKG_BUILD_DEPENDS` is now).

Reply:
Alright, I made that change in 62ea68f.

It does result in the systemd unit and the manpage getting installed, but I'm guessing that isn't a problem as long as they don't end up in the shipped binary (which they don't appear to).

Reply:
`Build/InstallDev` is meant to install headers and shared libraries (built for the target device) in a place where other packages can find and link to them, it's not a problem if other files are installed as well.